src/clients/locations/FeatureServiceClient.js

```diff
@@ -6,20 +6,28 @@ const trackDependency = require('../appinsights/AppInsightsClient').trackDepende
 
 const apiUrlBase = process.env.FORTIS_FEATURE_SERVICE_HOST;
 
-function formatIdsUri(ids) {
-  return `${apiUrlBase}/features/id/${ids.map(encodeURIComponent).join(',')}?include=bbox`;
+function formatIdsUri(ids, include) {
```

Comment:
FeatureServiceClient is not an object, these are all just functions. What we want to include depends on the context of the request (e.g. some callers need the bbox, others don't), so this really should be configurable on a per-call level.